fdbrpc/HealthMonitor.actor.cpp

```diff
@@ -36,6 +36,10 @@ void HealthMonitor::purgeOutdatedHistory() {
 			--count;
 			ASSERT(count >= 0);
 			peerClosedHistory.pop_front();
+
```

Comment:
@sfc-gh-yiwu It's not clear to me what these changes do or why they are needed.

Reply:
must be rebase problem. let me revert.